tools/check-schemas

```diff
@@ -61,6 +61,12 @@ def get_event_checker(event: dict[str, Any]) -> Callable[[str, dict[str, Any]],
     # Start by grabbing the event type.
     name = event["type"]
 
+    if name == "typing_edit_message":
+        if event["message_type"] == "stream":
+            name = "typing_edit_channel_message"
+        else:
+            name = "typing_edit_direct_message"
```

Comment:
This change is fine to me.  As mentioned in https://chat.zulip.org/#narrow/channel/43-automated-testing/topic/event.20definition.20in.20event_types.2Epy/near/2075556 we can fix this forward once this PR is merged.  I have a similar hack in one of my PRs, and we just need a few more samples before we decide on the better approach here.